lib/evaluation/handlers/limitDisclosureEvaluationHandler.ts

```diff
@@ -19,18 +20,39 @@ export class LimitDisclosureEvaluationHandler extends AbstractEvaluationHandler
 
   public handle(pd: PresentationDefinition, vcs: VerifiableCredential[]): void {
     pd.input_descriptors.forEach((inDesc: InputDescriptor, index: number) => {
-      if (inDesc.constraints?.fields && inDesc.constraints?.limit_disclosure === Optionality.Required) {
-        this.enforceLimitDisclosure(vcs, inDesc.constraints.fields, index);
+      if (
+        inDesc.constraints?.fields &&
+        (inDesc.constraints?.limit_disclosure === Optionality.Required ||
+          inDesc.constraints?.limit_disclosure === Optionality.Preferred)
+      ) {
+        this.enforceLimitDisclosure(vcs, inDesc.constraints, index);
       }
     });
   }
 
-  private enforceLimitDisclosure(verifiableCredential: VerifiableCredential[], fields: Field[], idIdx: number): void {
+  private limitDisclosureSupported(vc: VerifiableCredential, vcIdx: number, idIdx: number): boolean {
+    const limitDisclosureSignatures = LIMIT_DISCLOSURE_SIGNATURES;
```

Comment:
As discussed during sprint start. We need a constant list like we have now, but we also need the user to be able to configure which suites allow for selective disclosure , so they can add (future) suites

Reply:
Fixed!